validator/json_parser.py

```diff
@@ -0,0 +1,23 @@
+import json
+
+from validator.constants import FIREFOX_GUID
+
+
+class ManifestJsonParser():
```

Comment:
Because not inheriting from `object` creates an old-style class in Python 2 which, among a few other things causes `type` to give different results:

```python
>>> class Foo: pass
... 
>>> type(Foo())
<type 'instance'>
```

Compared to new-style classes:

```python
>>> class Bar(object): pass
... 
>>> type(Bar())
<class '__main__.Bar'>
```

Also they behave differently in multiple-inheritance scenarios. You should always prefer new-style classes which are the default in Python 3.